### PR TITLE
Support Kubernetes 1.20 (coreDNS 1.8.4) - HashiCorp Consul helm Chart

### PR DESCRIPTION
**Pull request by Shahard2**
The next flag got deprecated since k8s 1.20:

`service.alpha.kubernetes.io/tolerate-unready-endpoints: true`

so now we need to use:

`publishNotReadyAddresses: true`

if we don't change the flag, the consul pod will be in a crash loop with this errors:

`13:59:59.27 13:59:59.27 Welcome to the Bitnami consul container 13:59:59.27 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-consul 13:59:59.27 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-consul/issues 13:59:59.27 13:59:59.28 INFO ==> ** Starting Consul setup ** 13:59:59.30 INFO ==> Validating settings in CONSUL_* env vars.. 13:59:59.37 ERROR ==> The value for CONSUL_RETRY_JOIN_ADDRESS should be an IPv4 address or it must be a resolvable hostname`

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi,

Thank you so much for the PR! Could you bump the patch version of the chart?

**Comment by Shahard2** [comment]
> Hi,
> 
> Thank you so much for the PR! Could you bump the patch version of the chart?


Sure,
done mate

File: bitnami/consul/Chart.yaml [hunk 1]
```diff
@@ -3,24 +3,26 @@ annotations:
 apiVersion: v2
 appVersion: 1.10.1
 dependencies:
-  - name: common
-    repository: https://charts.bitnami.com/bitnami
-    tags:
-      - bitnami-common
-    version: 1.x.x
-description: Highly available and distributed service discovery and key-value store designed with support for the modern data center to make distributed systems and configuration easy.
+- name: common
```
Comment (javsalgar) [inline bitnami/consul/Chart.yaml:6]: It seems that you applied an auto-formatter incompatible with our yaml conventions. Could you fix it?

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: bitnami/consul/Chart.yaml [hunk 2]
```diff
@@ -4,23 +4,23 @@ apiVersion: v2
 appVersion: 1.10.1
 dependencies:
   - name: common
-    repository: https://charts.bitnami.com/bitnami
-    tags:
-      - bitnami-common
-    version: 1.x.x
+  repository: https://charts.bitnami.com/bitnami
```
Comment (javsalgar) [inline bitnami/consul/Chart.yaml:7]: The indentation is incorrect, check the original one

**Review by javsalgar — CHANGES_REQUESTED** [review]

**Review by javsalgar — APPROVED** [review]
Thank you! LGTM

**Pull request merged**